Renew physical card date specifications (#628)
swan-io/docs: docs/topics/cards/physical/index.mdx

| Card renewal status | Explanation |
|---|---|
| `ToRenew` | Triggered **10 weeks before the last day of the card's expiry month**, at midnight Coordinated Universal Time (UTC). This status schedules the renewal. You can stop the process by [canceling](./guide-cancel.mdx) the card, which permanently ends the renewal, or by [suspending](#renew-suspended) it, which temporarily pauses the process. The card won't be renewed if it remains `Suspended` past 8 weeks. |

> | `Renewed` | Triggered automatically **8 weeks before the last day of the expiry month**, at midnight UTC. At this point, the new card is created and queued for printing. No updates to the delivery address can be made after this status is triggered. Once the status changes to Renewed, the previous card's information is automatically available in the `previousPhysicalCards` field, even if the new card hasn't yet arrived or expired. |

Reviewer: ```suggestion
| `Renewed` | Triggered automatically **8 weeks before the last day of the expiry month**, at midnight UTC. At this point, the new card is created and queued for printing. No updates to the delivery address can be made after this status is triggered. You can query the new `Renewed` card's details (expiry date, masked number, identifier) via `statusInfo.newPhysicalCard` on `PhysicalCardRenewedStatusInfo`. |
```

The final sentence as written conflicts with two things:

- The existing "Activating the new card" section on the same page describes previousPhysicalCards as populated once the old card is marked isExpired=true, which happens on activation, not at the Renewed transition.
- Schema shape: PhysicalCard.previousPhysicalCards is "every previous Physical Card information", and PhysicalCard.identifier is "updated when a renewed card is activated", which suggests the current PhysicalCard record persists through renewal rather than moving itself into previousPhysicalCards. The field exposing the upcoming card's info at Renewed status is statusInfo.newPhysicalCard on PhysicalCardRenewedStatusInfo (non-nullable BasicPhysicalCardInfo).

Worth confirming exact behaviour with Vinicius before merging, but the current sentence shouldn't ship as-is given the contradiction with the section below it.

Author: Based on our Linear convo with Mia and Vini, I reverted back to the original text.

:::info Reference date for renewal checkpoints
Swan calculates the 10-week and 8-week checkpoints from the **last day of the card's expiry month**, not from the date printed on the card. 
For example, if a card's expiry date is `10/26`, Swan uses **31 October 2026** as the reference date.
Swan evaluates both checkpoints at **midnight UTC**. There is no grace period.
:::

### Suspended physical cards {#renew-suspended}

The renewal process for a suspended physical card depends on **when the card is resumed**, compared to two renewal checkpoints.
Swan evaluates both checkpoints at midnight UTC, calculated from the last day of the card's expiry month.

> - **10 weeks before expiry**: eligible cards move to `ToRenew`. If the card is `Suspended` at this checkpoint, it doesn't move to `ToRenew`, so it doesn't enter the renewal flow.
> - **8 weeks before expiry**: cards in `ToRenew` move to `Renewed`. Swan then creates and ships the new card. If the card is `Suspended` at this checkpoint, it doesn't move to `Renewed`.

Reviewer: ```suggestion
- **10 weeks before expiry**: eligible cards move to `ToRenew`. If the card is `Suspended` at this checkpoint, it doesn't move to `ToRenew`, so it doesn't enter the renewal flow.
- **8 weeks before expiry**: cards in `ToRenew` move to `Renewed`. Swan then creates and ships the new card. If the card is `Suspended` at this checkpoint, it doesn't move to `Renewed`.

Between these two checkpoints, you have a 2-week window to [update the delivery address](/topics/cards/physical/guide-renew).

```
Restores the 2-week update window statement from the previous version (lost actionable info for integrators planning their UX), preserving link to the renewal guide rather than naming a specific mechanism.

This doesn't affect the shipping of the new card, it will still be sent. 
You must resume the new card before it can be `Activated`.

#### Example scenario

Sofia Ramos has a physical card with an expiry date of `10/26`. The **reference date** is 31 October 2026 (the last day of the expiry month).

What happens at each checkpoint:

> 1. 10-week checkpoint (22 August 2026, midnight UTC)

Reviewer: ```suggestion
1. 10-week checkpoint (22 August 2026, midnight UTC).
```

- If the card is `Activated` or `ToActivate`, it moves to `ToRenew`.
- If the card is `Suspended`, it stays `Suspended` and doesn't enter the renewal flow.

> 2. 8-week checkpoint (5 September 2026, midnight UTC)

Reviewer: ```suggestion
2. 8-week checkpoint (5 September 2026, midnight UTC).
```
